packages/framework/src/Component/Domain/DomainSubscriber.php

```diff
@@ -39,7 +39,7 @@ public static function getSubscribedEvents(): array
     {
         return [
             // Setting domain by request must be done before loading other services (eg.: routing, localization...)
-            KernelEvents::REQUEST => [['onKernelRequest', 100]],
+            KernelEvents::REQUEST => [['onKernelRequest', 10000000]],
```

Comment:
_nitpick_ 
I find it hard to determine the priority with such many zeros.

what about setting some reasonable values for this and maintenanceModeSubscriber? 10000 and 20000 for instance